src/qt/rpcconsole.cpp

```diff
@@ -890,8 +888,10 @@ void RPCConsole::on_lineEdit_returnPressed()
 {
     QString cmd = ui->lineEdit->text();
 
-    if(!cmd.isEmpty())
-    {
+    if (!cmd.isEmpty()) {
+        ui->lineEdit->setEnabled(false);
```

Comment:
I think it's nice to be able to write the next command.

Reply:
Good point. What about allowing input but disallow executing (`enter` won't clear the line, etc.). The `executing...` text would need to be somewhere else though...

Reply:
@jonasschnelli and this https://github.com/bitcoin-core/gui/pull/123#pullrequestreview-519587704?

Reply:
Updated.